rotkehlchen/db/schema.py

```diff
@@ -810,6 +810,15 @@
 );
 """  # noqa: E501
 
+DB_CREATE_SOLANA_TX_MAPPINGS = """
```

Comment:
Why do we need a new table? Isn't it the same exact thing as evm transactions mappings except the tx_id references another table? Isn't there a better way of representing this in sqlite? Or a new table is the best way to do this? (not saying it's not -- just seems like duplicating logic )